src/main/java/ch/njol/skript/events/EvtBlock.java

```diff
@@ -74,6 +70,22 @@ public class EvtBlock extends SkriptEvent {
 			.description("Called when a block is created, but not by a player, e.g. snow forms due to snowfall, water freezes in cold biomes. This isn't called when block spreads (mushroom growth, water physics etc.), as it has its own event (see <a href='#spread'>spread event</a>).")
 			.examples("on form of snow:", "on form of a mushroom:")
 			.since("1.0, 2.6 (BlockData support)");
+		Skript.registerEvent("Block Drop", EvtBlock.class, BlockDropItemEvent.class, "block drop[ping] [[of] %-itemtypes/blockdatas%]")
+			.description(
+				"Called when a block broken by a player drops items",
+				"<ul>",
+				"<li>event-player : The player that broke the block</li>",
+				"<li>past event-block : The block that was broken</li>",
+				"<li>event-block : The block after being broken</li>",
+				"<li>event-items (or drops) : The drops of the block</li>",
+				"<li>event-entities : The entities of the dropped items</li>",
+				"</ul>",
+				"",
+				"If the breaking of the block leads to others being broken, such as torches, they will appear",
+				"in \"event-items\" and \"event-entities\"."
+			)
+			.examples("on block drop:", "on block drop of oak log:")
```

Comment:
should have at least some code in the event itself like broadcast, doesnt matter